sdcm/tester.py

```diff
@@ -163,6 +163,7 @@ def setUp(self):
         db_node_address = self.db_cluster.nodes[0].private_ip_address
         self.loaders.wait_for_init(db_node_address=db_node_address)
         nodes_monitored = [node.private_ip_address for node in self.db_cluster.nodes]
+        nodes_monitored += [node.private_ip_address for node in self.loaders.nodes]
```

Comment:
I believe other tests that have to implement their own setup procedures might need similar fixes @kongove.